intranet/static/css/dark/eighth.schedule.scss

```diff
@@ -1,6 +1,6 @@
 @media (min-width: 500px) {
     .schedule-activity-page.floating .eighth-header {
-        background-color: inherit;
+        background-color: black;
```

Comment:
This was merged in a PR a while ago. Can you try rebasing against the latest `dev` again?

Reply:
Oops, it seems like I merged one too many commits. I will try rebasing once again.